### PR TITLE
Add cluster-api presubmit to check for deps

### PR DESCRIPTION
**Pull request by alvaroaleman**
/assign @roberthbailey 

This requires an image with the changes in #9874 to be pushed and this PR to be updated with the new tag. Could you check if you have the permissions for that Robert?

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml [hunk 1]
```diff
@@ -50,3 +50,17 @@ presubmits:
         - "--scenario=execute"
         - "--"
         - "./scripts/ci-test.sh"
+  - name: pull-cluster-api-vendor-in-sync
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20181020-5ce477da2
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml]: v20190114-652973a54 now :-)

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml]: https://github.com/kubernetes/test-infra/pull/9874#issuecomment-454143375

Comment (alvaroaleman) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml]: Thank you!

**Comment by alvaroaleman** [comment]
@roberthbailey Pull should be good to go, PTAL

**Comment by roberthbailey** [comment]
Looks like @BenTheElder beat me to pushing an image. Thanks @BenTheElder!

/lgtm
/approve

**Pull request merged**